src/routes/home/__tests__/HomeScreen.spec.tsx

```diff
@@ -64,30 +67,17 @@ describe('HomeScreen', () => {
     expect(navigation.navigate).toHaveBeenCalledWith('DisciplineSelection', { discipline: mockCustomDiscipline })
   })
 
-  it('should delete custom discipline', async () => {
-    await AsyncStorageService.setCustomDisciplines(['test'])
-
-    mocked(useLoadDisciplines).mockReturnValueOnce(getReturnOf(mockDisciplines))
-    mocked(useReadCustomDisciplines).mockReturnValue(getReturnOf(['test']))
-    mocked(useLoadGroupInfo).mockReturnValue(getReturnOf(mockCustomDiscipline))
-    const spyOnDeletion = jest.spyOn(AsyncStorageService, 'deleteCustomDiscipline')
-
-    const { findByText, findByTestId } = render(<HomeScreen navigation={navigation} />)
-    const customDiscipline = await findByText('Custom Discipline')
-    expect(customDiscipline).toBeDefined()
-
-    const row = await findByTestId('Swipeable')
-    const swipeable = row.children[0] as ReactTestInstance
-    swipeable.instance.openRight()
+  it('should show suggestion to add custom discipline', async () => {
```

Comment:
i think there is no async needed instead you can use `getByText` without await. Same for the tests above